resources/dist/index.js

```diff
@@ -1,2 +1,2 @@
erencePreviewsEnabled:function(){return"0"!==p.get(y)},storeReferencePreviewsEnabled:function(e){e?p.remove(y):p.set(y,"0"),mw.track("event.ReferencePreviewsPopups",{action:e?"anonymousEnabled":"anonymousDisabled"})}}),_=function(e,t,n){return n.get("wgPopupsReferencePreviews")?n.get("wgPopupsConflictsWithRefTooltipsGadget")||"minerva"===n.get("skin")?null:e.isAnon()?t.isReferencePreviewsEnabled():!!n.get("wgPopupsReferencePreviews")||"1"===mw.user.options.get("popups-reference-previews")||null:null}(mw.user,x,mw.config),k=S(null!==_),C=(c=mw.experiments,{weightedBoolean:function(e,t,n){return"true"===c.getBucket({enabled:!0,name:e,buckets:{true:t,false:1-t}},n)}}),R=function(e,t,n){return function(e,t,n){var r=t.get("wgPopupsStatsvSamplingRate",0);return n.weightedBoolean("ext.Popups.statsv",r,e.sessionId())}(e,t,n)?mw.track:function(){}}(mw.user,mw.config,C),O=mw.config.get("wgPopupsVirtualPageViews")?mw.track:function(){},D=(t(n={},u.TYPE_PAGE,function(e,t,n){return mw.config.get("wgPopupsConflictsWithNavPopupGadget")?null:e.isAnon()?t.isPagePreviewsEnabled():"1"===mw.user.options.get("popups")||null}(mw.user,x)),t(n,u.TYPE_REFERENCE,_),n);mw.config.get("debug")&&(d=window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__||d);var L=r.createStore(r.combineReducers(Pe),d(r.applyMiddleware(o()))),H=r.bindActionCreators(e,L.dispatch),G=function(e,t){var n,r=function(){};return e.isAnon()?r=function(e){e.preventDefault(),t.showSettings()}:n=mw.Title.newFromText("Special:Preferences#mw-prefsection-rendering").getUrl(),{settingsUrl:n,showSettings:r,previewDwell:t.previewDwell,previewAbandon:t.abandon,previewShow:t.previewShow,click:t.linkClick}}(mw.user,H);!function(e,t,n,r,i,o,a){I(e,z(t)),I(e,Z()),I(e,Q(i)),I(e,te(t,o)),I(e,ne(n)),I(e,ee(t,r)),I(e,J(t,a))}(L,H,x,k,G,R,O),H.boot(D,mw.user,x,mw.config,window.location.href),mw.popups=function(e){return{isEnabled:function(){return e.getState().preview.enabled[u.TYPE_PAGE]}}}(L);var F,j=[];if(null!==D[u.TYPE_PAGE]){var Y=xe.join(", ");j.push("#mw-content-text a[href][title]:not(".concat(Y,")"))}if(null!==D[u.TYPE_REFERENCE]&&j.push('#mw-content-text .reference a[ href*="#" ]'),j.length){var W=j.join(", ");U()||(F=document.body,$("<div>").attr("id","mwe-popups-svg").html(N()).appendTo(F)),$(document).on("mouseover keyup",W,(function(e){var t=A(this,mw.config);if(t){var n,r=l(this,mw.config,t);switch(r){case u.TYPE_PAGE:n=v;break;case u.TYPE_REFERENCE:n=P;break;default:return}var i=$(this),o=$(window),a={pageX:e.pageX,pageY:e.pageY,clientY:e.clientY,width:i.width(),height:i.height(),offset:i.offset(),clientRects:this.getClientRects(),windowWidth:o.width(),windowHeight:o.height(),scrollTop:o.scrollTop()};H.linkDwell(t,this,a,n,m,r)}})).on("mouseout blur",W,(function(){A(this,mw.config)&&H.abandon()})).on("click",W,(function(){var e=A(this,mw.config);e&&u.TYPE_PAGE===l(this,mw.config,e)&&H.linkClick(this)}))}else mw.log.warn("ext.popups was loaded but everything is disabled")}(),window.Redux=r,window.ReduxThunk=i}()}();
```

Comment:
Was the update to these 2 pages intentional?

Reply:
Yes, the popups extension uses webpack to build the modules that are actually sent to the user, see the documentation in the README